docs/concepts/model-training.md

```diff
@@ -0,0 +1,34 @@
+(model-training)=
+# Model Training
+
+There are many methods and techniques for training a machine learning model. However, there is much more to model training than the training code itself.
+
+MLRun provides MLOps functionality for model training including job orchestration, experiment tracking, creating reusable components, and distributed training out of the box. Please see below for information on each of these capabilies:
+
+### Jobs
+
+Training a model is an example of a `Job` - something that is run once to completion. MLRun provides you with a convenient syntax to take a Python training script and automatically package and deploy on top of a production ready Kubernetes cluster. You can also configure many aspects of this deployment including CPU/MEM/GPU resources, Python dependencies, distributed runtimes, and more.
+
+See the [Create a Basic Training Job](#) page for an example of a simple training job. Additionally, see [Managing Job Resources](https://docs.mlrun.org/en/latest/runtimes/configuring-job-resources.html) for ways to configure your jobs.
+
+### Logging Artifacts
+
+While training your model, there may be things you want to log including the model itself, datasets, plots/charts, metrics, etc. All of this and more can be tracked using MLRun experiment tracking.
+
+There is support for automatic logging for major ML frameworks such as sklearn, PyTorch, TensorFlow, LightGBM, etc. There is also support for manually logging models, datasets, metrics, and more.
```

Comment:
MLRun supports automatic logging for major ML frameworks such as sklearn, PyTorch, TensorFlow, LightGBM, etc. MLRun also supports manually logging models, datasets, metrics, and more.